README.md

```diff
@@ -57,6 +57,13 @@ The SpeedUpAmerica project utilizes the following technologies for operation:
 
 These instructions work on Linux, Windows and MacOS and only need to be performed once, unless you reset your database or config files.
 
+Install [Git](https://git-scm.com/downloads) Windows/Mac/Linux
+> If you already have Git installed on your machine, you can update to the lastest version via your CMD:
```

Comment:
This is a heavy-handed way to upgrade git that will require additional steps and is outside of the scope of or project. We just need them install `git`, how is up to them. Please remove Lines 61-65.

Reply:
Leave the `Install ...` line. :)

Reply:
Sounds good! I will remove those lines.